users/config.rst

```diff
@@ -591,6 +591,10 @@ Listen Addresses
 
 The following address types are accepted in sync protocol listen addresses:
 
+Default listen addresses (``default``)
+    This is equivalent to ``tcp://0.0.0.0:22000`` and
```

Comment:
What does "and" mean here? How do I put both of these in the listen address box?